src/main/java/org/passay/PolishSequenceData.java

```diff
@@ -0,0 +1,79 @@
+/* See LICENSE for licensing and NOTICE for copyright. */
+package org.passay;
+
+/**
+ * Polish character sequences.
+ *
+ * @author  Middleware Services + Wojciech Buczko
+ */
+public enum PolishSequenceData implements SequenceData {
+
+    /** Alphabetical sequence. */
+    Alphabetical(
+            "ILLEGAL_ALPHABETICAL_SEQUENCE",
+            new CharacterSequence[] {new CharacterSequence("abcdefghijklmnopqrstuvwxyz", "ABCDEFGHIJKLMNOPQRSTUVWXYZ"),
+                    new CharacterSequence("aąbcćdeęfghijklmnoópqrstuvwxyzźż", "AĄBCĆDEĘFGHIJKLMNOÓPQRSTUVWXYZŹŻ"), }),
```

Comment:
Did you intentionally include the English CharacterSequence data here?

Reply:
I believe it is better to use multiple rules and be explicit:
```
List<Rule> rules = new ArrayList<>();
rules.add(new IllegalSequenceRule(EnglishSequenceData.Alphabetical));
rules.add(new IllegalSequenceRule(PolishSequenceData.Alphabetical));
```

See 72621c058b096b209418aaf70e7e805c0bb6bf2e